doc/source/getting-started.rst

```diff
@@ -100,6 +100,24 @@ With that configuration, pull requests sent to the branch auto-deployment
 require no approvals, while still requiring Travis continuous integration to
 succeed.
 
+You can also disable Mergify on a particular branch:
+
+.. code-block:: yaml
+
+    rules:
+      protection:
+        required_status_checks:
+          contexts:
+            - continuous-integration/travis-ci
+        required_pull_request_reviews:
+          required_approving_review_count: 2
+      branches:
+        my-feature-branch: null
+
+Here the branch my-feature-branch will not have Mergify enabled. All pull requests
```

Comment:
Maybe put backquotes or something around the branch name so it show differently?